connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -216,7 +217,10 @@ Set<String> findConsumerGroups()
 
     Collection<ConsumerGroupListing> listConsumerGroups()
             throws InterruptedException, ExecutionException {
-        return sourceAdminClient.listConsumerGroups().valid().get();
+        return adminCall(
+                () -> sourceAdminClient.listConsumerGroups().valid().get(),
+                () -> "list consumer groups on cluster " + config.sourceClusterAlias()
```

Comment:
Should we put the alias before `cluster`? so we get a message like `list consumer groups on source cluster` for example. I think it reads better than `list consumer groups on cluster source`

Reply:
Done.

Reply:
I don't see the change, did you forget to push?

Reply:
Yes, it's ready for review now.